src/Symfony/Component/Finder/Iterator/SortableIterator.php

```diff
@@ -76,7 +86,8 @@ public function __construct(\Traversable $iterator, int|callable $sort, bool $re
         } elseif (self::SORT_BY_NONE === $sort) {
             $this->sort = $order;
         } elseif (\is_callable($sort)) {
-            $this->sort = $reverseOrder ? static function (\SplFileInfo $a, \SplFileInfo $b) use ($sort) { return -$sort($a, $b); } : $sort(...);
+            $this->sort = $reverseOrder ? static function (\SplFileInfo $a, \SplFileInfo $b) use ($sort) { return -$sort($a, $b); }
+            : $sort(...);
```

Comment:
this change should be reverted

Reply:
I did revert the change but fabbot.io did not like it.

Reply:
this looks like a bug in php-cs-fixer to me. There is no reason for this to go in the next line in our coding standards.

Reply:
is there a max character per line limit ?

Reply:
Not in the Symfony coding standards